doc/install_ubuntu.txt

```diff
@@ -144,13 +164,22 @@ system package, you can run this:
 Bleeding edge
 ~~~~~~~~~~~~~
 
-Do like in the section "Updating Theano", but use
-``git+git://github.com/Theano/Theano.git`` instead of ``theano``.
+Change to the Theano directory and run:
+
+.. code-block: bash
+    
+    git pull
+    python setup.py develop --user
 
 
 Manual Openblas instruction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. comment::
+
+    I believe this is outdated, my machine seems to be using 8 threads 
+    happily with the binary openblas...
+
```

Comment:
With ubuntu 12.04, on my laptop with 4 core, there is only 2 cores used. So this could depend of the version of the OS.